### PR TITLE
Update Readme.md

### PR DESCRIPTION
**Pull request by amanregu**

### CODE REVIEW

**Review by PrateekxCodes — CHANGES_REQUESTED** [review]
@singhabhinav96 @amanregu Please update `Wiki Links` to `Wiki`

**Review by PrateekxCodes — APPROVED** [review]
Looks fine!

File: README.md [hunk 1]
```diff
@@ -1 +1,35 @@
-Orbit
+# Orbit - Automating AltCampus application process
+
+## About Orbit
+This application aims to smoothen the applicant  application process at AltCampus. Users can login to see their application progress in real time. Users can download and submit assignments, solve questions and schedule an interview. Admins can grade the submited assignments and questions. Based on the score(both the objectively calculated and the one assigned by the admins) it will be determined whether the user is qualified for the interview or not.
```
Comment (prank7) [inline README.md]: "smoothen the applicant  application process" doesn't seem right. 

Break the paragraph into two paras. After "Admins can ...." should be another paragraph.

Please fix it.

**Review by prank7 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (prank7) [reply README.md]: @amanregu @singhabhinav96 please check the review and make fixes.

Comment (abhinavsingh06) [reply README.md]: @prank7 Updated as per the requirement. Please review.

**Review by PrateekxCodes — APPROVED** [review]
All Looks good!

**Comment by prank7** [comment]
Looks good. Closes #4

**Pull request merged**